docs/docusaurus/versioned_docs/version-1.5.0/lte/debug_dp_probe.md

```diff
@@ -11,29 +11,48 @@ The datapath probe CLI helps an operator to probe the Datapath to the UE.
 
 ## Usage
 
+On your GW host, run the following command
+
+```$ dp_probe_cli.py -i <IMSI> --direction <DL/UL> list_rules```
```

Comment:
can you add example of --list_rules execution?

Reply:
Hey @pshelar I added in line 52 of this file. Same on the master doc file

```sh
dp_probe_cli.py -i 1010000051013 --direction UL list_rules
IMSI: 1010000051013, IP: 192.168.128.13
Uplink rules: allowlist_sid-IMSI1010000051013-magma.ipv4
```